src/WebJobs.Extensions.DurableTask/OrchestrationClientAttribute.cs

```diff
@@ -1,10 +1,9 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using Microsoft.Azure.WebJobs.Description;
```

Comment:
Minor stylistic quibble: System* using statements should appear before any others.